test/test_helper.rb

```diff
@@ -26,12 +26,12 @@ def set_organization(organization)
     Rails.application.routes.default_url_options[:script_name] = "/#{organization.slug}"
   end
 
-  def setup
-    ActsAsTenant.current_tenant = create(:organization, slug: "test")
+  setup do
```

Comment:
The issue was that `def setup` is called after the `setup` blocks in individual tests:

```
# test/test_helper.rb
def setup
  puts "I'm in test_helper.rb!"
end

# test/some_dummy_test.rb
setup do
  puts "I'm in dummy_test.rb!"
end

# > I'm in dummy_test.rb!
# > I'm in test_helper.rb!
```

By converting it to `setup do; ... end` we fix the order issue :) 

```
# test/test_helper.rb
setup do
  puts "I'm in test_helper.rb!"
end

# test/some_dummy_test.rb
setup do
  puts "I'm in dummy_test.rb!"
end

# > I'm in test_helper.rb!
# > I'm in dummy_test.rb!
```

This is important because we create an organization in this setup and assign it as current test tenant.